### ISSUE TITLE
When running CPUID memprotect or memmove sometimes fails 

### ISSUE THREAD

**Issue by workhorsy** [workhorsy/py-cpuinfo#82]
For some reason, running CPUID sometimes fails on some variation of OS, Python version, virtual machine, and pyinstaller. It usually fails when trying to memprotect or memmove.

windows 8 x86_64, python 2, vm: ok
windows 8 x86_64, python 2, vm, pyinstaller: ok
windows 8 x86_64, python 3, vm: **FAIL** (line 611 OSError: exception: access violation writing 0xFFFF8768767)
windows 8 x86_64, python 3, vm, pyinstaller: ok

fedora 24 x86_64, python 2, vm: ok
fedora 24 x86_64, python 3, vm: ok

ubuntu 16.10 x86_64, python 2, vm: **FAIL**
ubuntu 16.10 x86_64, python 2, vm, pyinstaller: ok
ubuntu 16.10 x86_64, python 3, vm: **FAIL** (line 624 fails on ctypes.pythonapi.mprotect)

ubuntu 16.04 x86_64, python 2: ok
ubuntu 16.04 x86_64, python 3: ok

**Comment by workhorsy** [comment]
This seems to only happen intermittently on Linux now.
```
c_ulong(8L)
c_void_p(94063251230720)
Process Process-1:
Traceback (most recent call last):
  File "/usr/lib/python2.7/multiprocessing/process.py", line 267, in _bootstrap
    self.run()
  File "/usr/lib/python2.7/multiprocessing/process.py", line 114, in run
    self._target(*self._args, **self._kwargs)
  File "cpuinfo/cpuinfo.py", line 1225, in _actual_get_cpu_info_from_cpuid
    max_extension_support = cpuid.get_max_extension_support()
  File "cpuinfo/cpuinfo.py", line 777, in get_max_extension_support
    b"\xB8\x00\x00\x00\x80" # mov ax,0x80000000
  File "cpuinfo/cpuinfo.py", line 674, in _run_asm
    func, address = self._asm_func(restype, argtypes, byte_code)
  File "cpuinfo/cpuinfo.py", line 657, in _asm_func
    raise Exception("Failed to memmove")
Exception: Failed to memmove
```

**Comment by workhorsy** [comment]
Looks like this was fixed by pr #84 . Closing now.
